Software/Development/SemesterProject/App/Include/referenceFrameConversion.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * referenceFrameConversion.h
+ *
+ *  Created on: Apr 9, 2019
+ *      Author: Nicolas
+ */
+
+#ifndef APP_INCLUDE_REFERENCEFRAMECONVERSION_H_
+#define APP_INCLUDE_REFERENCEFRAMECONVERSION_H_
+
+
+/*
+ * Temporary declarations To be removed
+ */
+#include "analogAcquisitionManager.h"
+void runClosedLoopControl(void);
+/*
+ * end of temporaty declarations
+ */
+
+
+typedef struct
+{
+    float DComponent;
+    float QComponent;
+}DQObject;
+
+typedef struct
+{
+    float alphaComponent;
+    float betaComponent;
+}alphaBetaObject;
```

Comment:
I think you should consider using afaBeta 🤔

Reply:
( ͡° ͜ʖ ͡°)